spock-container-test-app/gradle.properties

```diff
@@ -1,6 +1,8 @@
-grailsVersion=6.0.0
-grailsGradlePluginVersion=6.0.0
 version=0.1
+
+grailsVersion=6.2.2
```

Comment:
For the backport, I would choose the minimum grails version so people know the lowest level of grails we are compatible with

Reply:
This is the test app. The plugin should be compatible with all Grails 6 versions.

Reply:
choosing the minimum version in the test app could show that 😀

Reply:
Yes, you could see it that way. In this case, until proven wrong, I chose to lean on semantic versioning rules :smile: 

Do you want me to change it back to `6.0.0`? (I have tested it with `6.0.0`, and it works)

Reply:
If you have tested with it, I am good.  I hope we don't have that many more 6.x releases.